core/domain/exp_domain.py

```diff
@@ -375,11 +375,11 @@ def validate(self):
             raise utils.ValidationError(
                 'Invalid audio filename: %s' % self.filename)
         extension = self.filename[dot_index + 1:]
-        if extension not in feconf.ACCEPTED_AUDIO_EXTENSIONS:
+        if extension not in feconf.ACCEPTED_AUDIO_EXTENSIONS.keys():
```

Comment:
I don't think it does since I changed feconf.ACCEPTED_AUDIO_EXTENSION from a list into a dict, and the existing values into keys